apps/pm-user/src/components/atoms/formWraper/FormWraper.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styles from './formWraper.module.scss';
+import PropTypes from 'prop-types';
+import Logo from '../logo';
+import { Link } from 'react-router-dom';
+
+const FormWraper = ({
+  bottomText,
+  children,
+  formDescription,
+  formTitle,
+  pageToRedirect,
+  pageToRedirectTitle,
+  sideImage,
+}) => {
+  return (
+    <div className={styles.superContainer}>
+      <div className={styles.leftContainer}>
+        <img src={require(`../../../assets/images/${sideImage}.jpg`)} />
+      </div>
+      <div className={styles.rightContainer}>
+        <div className={styles.loginBox}>
+          {/* <div className={styles.logInLogo}>Pesto Matrimony</div> */}
+          <Logo size="medium" />
+          <div className={styles.formTitle}>{formTitle}</div>
+          <div className={styles.formDescription}>{formDescription}</div>
+          {children}
+          <p className={styles.formBottomText}>
+            {bottomText}
+            <Link to={`/${pageToRedirect}`}>
+              <span className={styles.redirectLink}>{pageToRedirectTitle}</span>
+            </Link>
+          </p>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+FormWraper.propTypes = {
+  bottomText: PropTypes.string,
+  children: PropTypes.object,
+  formTitle: PropTypes.string,
+  pageToRedirect: PropTypes.string,
+  pageToRedirectTitle: PropTypes.string,
+  sideImage: PropTypes.string,
+};
+
+FormWraper.defaultProps = {
+  bottomText: 'already have account then!',
```

Comment:
Better to say "Already have an account ?" instead of above text in my opinion.

Reply:
And why make this a prop. We wont be reusing it anywhere else.

Reply:
using it in the login page